addons/website_links/tests/test_ui.py

```diff
@@ -20,7 +20,7 @@ def _get_title_from_url(addr, **kw):
     def test_01_test_ui(self):
         self.env['link.tracker'].search_or_create({
             'campaign_id': self.env['utm.campaign'].create({'name': 'Sale'}).id,
-            'medium_id': self.env['utm.medium'].create({'name': 'Website'}).id,
+            'medium_id': self.env['utm.medium'].create({'name': 'Test Website'}).id,
             'source_id': self.env['utm.source'].create({'name': 'Search'}).id,
```

Comment:
@reth-odoo can you please review it again? especially this file. maybe i can also ask to tde? this file didn't make much sense to me, because: why do we create search, when we will only use search engine.

Reply:
@reth-odoo i will answer and read review later, but do you know the answer for this: why do we create search here and then later in a js file we invoke search engine. I think js assumes that demo data is loaded, (otherwise how else is it going to retrieve search engine)

Reply:
@datonefaridze indeed, I guess this was "fixed" to acomodate demo data as `like` would match any source with "search" in it, I guess with the lowest id first.

Arguably, this is a bug and we should match exactly the name, not anything that contains the name.

Good catch :)
Maybe check with Luc if we want to change that, and if we want to change it in stable

Reply:
@datonefaridze nevermind the previous comment, am dumb :monkey: 

Indeed we actively select "Search engine" in the js test, instead of "Search" for some reason. This test is weird either way though :raised_eyebrow: 

Would be good to fix it to select "Search" indeed (or I guess "search" now)